smoker/client/cli.py

```diff
@@ -7,7 +7,9 @@
 
 See pydoc for command line tool smoker.py
 """
+from __future__ import print_function
 
```

Comment:
Do these two imports work the same in Python 2/3?

Reply:
To my knowledge, yes:). The future-import ensures that the Python2 print works as a function (just like in Python3) instead of mistakingly interpreting the arguments separated by a comma inside the brackets as a tuple. In Python3, the print should work the same as always (as a function, that is).